lzy/allocator/src/test/java/ai/lzy/allocator/test/AllocatorServiceMountsTest.java

```diff
@@ -0,0 +1,150 @@
+package ai.lzy.allocator.test;
+
+import ai.lzy.allocator.alloc.dao.DynamicMountDao;
+import ai.lzy.allocator.alloc.dao.VmDao;
+import ai.lzy.allocator.model.DiskVolumeDescription;
+import ai.lzy.allocator.model.DynamicMount;
+import ai.lzy.allocator.model.Vm;
+import ai.lzy.v1.VmAllocatorApi;
+import ai.lzy.v1.VolumeApi;
+import com.google.protobuf.util.Durations;
+import io.grpc.Status;
+import io.grpc.StatusRuntimeException;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.io.IOException;
+import java.util.Map;
+import java.util.UUID;
+import java.util.stream.Collectors;
+
+@SuppressWarnings("ResultOfMethodCallIgnored")
+public class AllocatorServiceMountsTest extends AllocatorApiTestBase {
+
+    private VmDao vmDao;
+    private DynamicMountDao dynamicMountDao;
+
+    @Before
+    public void before() throws IOException {
+        super.setUp();
+
+        vmDao = allocatorCtx.getBean(VmDao.class);
+        dynamicMountDao = allocatorCtx.getBean(DynamicMountDao.class);
+    }
+
+    @After
+    public void after() {
+        super.tearDown();
+    }
+
+    @Override
+    protected void updateStartupProperties(Map<String, Object> props) {
+        super.updateStartupProperties(props);
+        props.put("allocator.allocation-timeout", "30s");
+        props.put("allocator.kuber-tunnel-allocator.enabled", "true");
+        props.put("allocator.mount.enabled", "true");
+        props.put("allocator.mount.podImage", "ubuntu");
+        props.put("allocator.mount.workerMountPoint", "/mnt/worker");
```

Comment:
let's move all temp data to the `'/tmp` dir

Reply:
These are just stubs with no meaning. These values won't be used to actually mount something in test env